config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
-root 'users#index'
-  resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
-  end
+  
+  
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root'users#index'
+
+  resources :users, only: [:index, :show, :new, :create] do
+    resources :posts, only: [:index, :new, :show, :create] do
+      resources :comments, only: [:new, :create]
```

Comment:
Please add a route for creating a `like` here. We can not like posts without a route to the Likes controller.

Reply:
resolved